src/components/Widget/style.scss

```diff
@@ -9,7 +9,6 @@
   max-width: 370px;
   position: fixed;
   right: 0;
-  width: 90vw;
   z-index: 9999;
 
```

Comment:
With your changes this is the result on my phone (iPhone 7)
![img_2740](https://user-images.githubusercontent.com/6841201/36264849-282d96cc-126e-11e8-9374-aa34e1991021.PNG)

In order to fix this bug I added:
`max-height: 100%;`
In `.widget-container {...`. Here is the result:

![img_2741](https://user-images.githubusercontent.com/6841201/36264851-2ae6dc84-126e-11e8-84db-85d09e5e2311.PNG)